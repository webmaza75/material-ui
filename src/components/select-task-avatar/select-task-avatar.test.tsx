import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import SelectTaskAvatar from './select-task-avatar';

describe(`SelectTaskAvatar`, () => {
  it(`TaskAvatar renders correctly with selected item`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const isSelectedItem = true;
    const result = renderer
      .render(<SelectTaskAvatar isSelectedItem={isSelectedItem} />);

    expect(result).toMatchSnapshot();
  });

  it(`TaskAvatar renders correctly with non-selected item`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const isSelectedItem = false;
    const result = renderer
      .render(<SelectTaskAvatar isSelectedItem={isSelectedItem} />);

    expect(result).toMatchSnapshot();
  });
});
