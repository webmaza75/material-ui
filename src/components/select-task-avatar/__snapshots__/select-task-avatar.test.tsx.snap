// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectTaskAvatar TaskAvatar renders correctly with non-selected item 1`] = `
<WithStyles(ForwardRef(Avatar))
  style={
    Object {
      "backgroundColor": "white",
      "border": "1px solid #b2ebf2",
      "color": "#80deea",
      "fontSize": 20,
      "height": 42,
      "lineHeight": 42,
      "margin": 12,
      "width": 42,
    }
  }
/>
`;

exports[`SelectTaskAvatar TaskAvatar renders correctly with selected item 1`] = `
<WithStyles(ForwardRef(Avatar))
  style={
    Object {
      "backgroundColor": "white",
      "border": "1px solid #b2ebf2",
      "color": "#80deea",
      "fontSize": 20,
      "height": 42,
      "lineHeight": 42,
      "margin": 12,
      "width": 42,
    }
  }
>
  <Memo(DoneIcon) />
</WithStyles(ForwardRef(Avatar))>
`;
